### PR TITLE
[BXMSPROD-2063] add kogito-images project as part of .ci/nightly-build-config.yaml

### PR DESCRIPTION
**Pull request by rgdoliveira**
Jira: https://issues.redhat.com/browse/BXMSPROD-2063

### CODE REVIEW

File: .ci/nightly-project-dependencies.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+      - project: kiegroup/kogito-examples
+    mapping:
+      dependencies:
+        default:
+          - source: (\d*)\.(.*)\.(.*)
+            targetExpression: "process.env.GITHUB_BASE_REF.replace(/(\\d*)\\.(.*)\\.(.*)/g, (m, n1, n2, n3) => `${+n1+7}.${n2}.${n3}`)"
+      dependant:
+        default:
+          - source: (\d*)\.(.*)\.(.*)
+            targetExpression: "process.env.GITHUB_BASE_REF.replace(/(\\d*)\\.(.*)\\.(.*)/g, (m, n1, n2, n3) => `${+n1-7}.${n2}.${n3}`)"
```
Comment (radtriste) [inline .ci/nightly-project-dependencies.yaml]: You should not need that mapping. kogito projects are aligned in version

Comment (Ginxo) [inline .ci/nightly-project-dependencies.yaml]:
```suggestion
```

**Review by Ginxo — CHANGES_REQUESTED** [review]
just remove mapping as suggested

File: .ci/nightly-project-dependencies.yaml (hunk 1 above)
Comment (rgdoliveira) [reply .ci/nightly-project-dependencies.yaml]: done, thanks!

**Review by lampajr — APPROVED** [review]
LGTM

**Review by radtriste — APPROVED** [review]

**Review by bacciotti — APPROVED** [review]

**Comment by rgdoliveira** [comment]
Backport PR https://github.com/kiegroup/kogito-pipelines/pull/996 merged into branch `1.40.x`

**Pull request merged**